src/core/bpf-devices.c

```diff
@@ -252,20 +252,9 @@ int bpf_devices_apply_policy(
 }
 
 int bpf_devices_supported(void) {
-        const struct bpf_insn trivial[] = {
-                BPF_MOV64_IMM(BPF_REG_0, 1),
-                BPF_EXIT_INSN()
-        };
-
-        _cleanup_(bpf_program_freep) BPFProgram *program = NULL;
         static int supported = -1;
-        int r;
 
-        /* Checks whether BPF device controller is supported. For this, we check two things:
-         *
-         * a) whether we are privileged
-         * b) the BPF implementation in the kernel supports BPF_PROG_TYPE_CGROUP_DEVICE programs, which we require
```

Comment:
Please list this in the README

Reply:
It is already there?

Reply:
my bad, I grepped for BPF_PROG_TYPE_CGROUP_DEVICE and missed it

Reply:
Already listed. Please see the line for v4.15.